Dockerfile

```diff
@@ -7,8 +7,18 @@ RUN \
     cd /src && \
     CGO_ENABLED=0 go build -ldflags "-extldflags -static -s" -o /steve
 
-FROM registry.suse.com/bci/bci-micro:15.5.11.2
+FROM registry.suse.com/bci/bci-micro:15.5
+
+ARG user=steve
+
+RUN echo "$user:x:1000:1000::/home/$user:/bin/bash" >> /etc/passwd && \
+    echo "$user:x:1000:" >> /etc/group && \
+    mkdir /home/$user && \
+    chown -R $user:$user /home/$user
+
 COPY --from=build /steve /usr/bin/steve
 # Hack to make golang do files,dns search order
 ENV LOCALDOMAIN=""
+USER $user
+WORKDIR /home/$user
```

Comment:
(nit) I don't think WORKDIR is needed, since there are no subsequent commands that use it as a relative path.

Reply:
Fixed!